src/control/cmd/daos_server/storage_nvme.go

```diff
@@ -76,34 +79,59 @@ func updatePrepReqParams(log logging.Logger, iommuEnabled bool, req *storage.Bde
 		req.EnableVMD = true
 	}
 
+	// Commandline PCI address lists will be comma-separated, sanitize into expected format.
+
+	if strings.Contains(req.PCIAllowList, " ") {
+		return errors.New("expecting comma-separated list of allowed pci addresses but found space separator")
+	}
```

Comment:
This seems unnecessarily hostile... If a space separator introduced ambiguity I could understand it, but otherwise it seems like it would friendlier and not risky to just go ahead on split on whitespace. If nothing else, the error message should just be a more generic "invalid input" error instead of "we know what you meant, but you didn't do it exactly the way we wanted you to so nyaah".

Reply:
yes I can do this, the positional arguments could be quoted with whitespace separators in the case of the allow list
can I please make the change in the follow-up PR: https://github.com/daos-stack/daos/pull/10022 so that we can make progress and get this one landed

Reply:
done in PR-10022